src/clr-angular/data/datagrid/datagrid-column-separator.ts

```diff
@@ -0,0 +1,70 @@
+/*
+ * Copyright (c) 2016-2018 VMware, Inc. All Rights Reserved.
+ * This software is released under MIT license.
+ * The full license information can be found in LICENSE in the root directory of this project.
+ */
+import { Component, Renderer2 } from '@angular/core';
+
+import { ColumnResizerService } from './providers/column-resizer.service';
+import { TableSizeService } from './providers/table-size.service';
+
+let nbCount: number = 0;
+
+@Component({
+  selector: 'clr-dg-column-separator',
+  template: `
+    <button class="datagrid-column-handle" tabindex="-1" type="button"
```

Comment:
@Jinnie I think your question regarding whether this should be a button or not was a good inquiry. I don't find strong compelling reasons for it to be a button element. I am still contemplating on the question. Hence, I am open to ideas and suggestions to answer the question.

Reply:
I agree that things which are clickable, and lead to some actions, should have the corresponding semantics. But in this specific case, the column separator is only for updating the visual display of some data. I even think we should deliberately try to keep it out of the screenreaders' radars.

Reply:
I agree with @Jinnie, it can be just a `div`. There is simply no benefit for a screen reader to resize columns, the content is not visible. In other cases, I did see `aria-grabbed` for draggable operations.

Reply:
What about low-mobility users that use a regular browser but keyboard navigation? They don't get to resize the columns and they can't read the actual cells content?

Reply:
Shouldn't columns always be displaying their full content by default? If that isn't the case, then it would be important to support keyboard column resizing (though the behaviors are something to think through more). We don't have keyboard support built in for this yet, so this seems beyond scope for the current capability.